src/zac/core/views/zaken.py

```diff
@@ -299,3 +307,73 @@ def get_object(self):
         zaak = find_zaak(**self.kwargs)
         self.check_object_permissions(zaak)
         return zaak
+
+
+class AccessRequestCreateView(LoginRequiredMixin, FormView):
+    form_class = AccessRequestCreateForm
+    template_name = "core/create_access_request.html"
+    #  todo add thanks page?
+    success_url = reverse_lazy("core:index")
+
+    def get_zaak(self):
+        return find_zaak(**self.request.resolver_match.kwargs)
```

Comment:
```suggestion
        return find_zaak(**self.kwargs)
```
Bit more consistent with the other views and this uses the top-level API instead of request resolver internals.